src/main/scala/cognite/spark/v1/RawTableRelation.scala

```diff
@@ -122,12 +122,18 @@ class RawTableRelation(
   override def buildScan(): RDD[Row] = buildScan(Array.empty, Array.empty)
 
   override def buildScan(requiredColumns: Array[String], filters: Array[Filter]): RDD[Row] = {
-    val (minLastUpdatedTime, maxLastUpdatedTime) = filtersToTimestampLimits(filters, "lastUpdatedTime")
+    if (requiredColumns.mkString(",").length > 200) {
+      // Temp fix until this parameter is moved into the request body
+      throw new CdfSparkIllegalArgumentException("Fetching a large number of columns is not supported")
+    }
```

Comment:
This can actually just be part of the if-statement below: If they specify a lot of columns, and exceed length of 200, we will just fetch all the data like normally, and Spark will filter the columns afterwards.. a little slower than optimal, but better than throwing exception :)